server/controllers/auth.js

```diff
@@ -9,11 +10,19 @@ authRouter.get("/google", passport.authenticate("google", { scope: ["profile", "
 // @route GET /auth/google/callback
 authRouter.get("/google/callback", passport.authenticate("google", { failureRedirect: "/" }), 
     (request, response) => {
-        response.json({
-            "status":"success",
-            "message": "user authenticated!"
-        }).end();
+        // response.json({
+        //     "status":"success",
+        //     "message": "user authenticated!"
+        // }).end();
+        // response.send(request.user);
+        response.redirect("http://localhost:3000/profile");
     }
 );
 
+// @desc Get user data
+authRouter.get("/users/me", (request, response) => {
+    console.log(request.user);
```

Comment:
is this also returning null? if so - i don't think `request.user` exists, i'm guessing you may expect this from the auth middleware but this does not have a middleware in this router

Reply:
https://stackoverflow.com/questions/62550277/passport-google-authentication-non-persistent 

in addition, try console.logging `request.session.passport` here and see what you get, from documentation this may be where your user info being stored

Reply:
hmm, yeah I'm getting a status error that user doesn't exist. let me check that link you sent and see if I can fix it.